src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -701,8 +701,8 @@ function copyOptions() {
 
     // Number of recipient setup
     formatNumberBox($currRecipient.val(), NEW_QUESTION);
-    var $prevRadioButtons = $('table[class~="questionTable"]').eq(-2).find('input[name="numofrecipientstype"]');
-    var $currRadioButtons = $('table[class~="questionTable"]').last().find('input[name="numofrecipientstype"]');
+    var $prevRadioButtons = $('div[class~="questionTable"]').eq(-2).find('input[name="numofrecipientstype"]');
```

Comment:
Since we are here, can we make this more future proof? I understand that adding tests to cover this case will ensure that it doesn't break. We still want to make sure that changing the HTML structure of this page in the future does not consistently result in extra work for the maintainer